app/views/users/index.html.erb

```diff
@@ -1,6 +1,5 @@
 <%  @users.each  do |user| %>
-<a href="/users/<%= user.id %>" class="link" >
-<%= render partial: "user_card", object: user, as: 'user' %>
-</a>
-<% end %>
-
+    <a href="/users/<%= user.id %>" class="link" >
+    <%= render partial: "user_card", object: user, as: 'user' %>
+    </a>
+    <% end %>
```

Comment:
- [ ] Kindly, fix this issue that is happening inside the ` app/views/users/index.html.erb`. before you request another review.

![blogform](https://user-images.githubusercontent.com/55635977/149089579-43af0cd4-eea7-4f48-9b3f-c130669b6a36.png)

Reply:
Fixed